src/ufdata.cc

```diff
@@ -73,35 +73,24 @@ Obj UF_TABLE(Obj self, Obj ufdata) {
   return gap_table;
 }
 
-// FIXME it is not valid to store Obj's inside a C++ container such as a vector
-
 Obj UF_BLOCKS(Obj self, Obj ufdata) {
   UFData::blocks_t* blocks = CLASS_OBJ<UFData*>(ufdata)->get_blocks();
```

Comment:
This could be `UFData::block_t const*` since  `blocks` is not modified by this function.